Documentation/security/policy/language.rst

```diff
@@ -659,6 +659,8 @@ port. When the application makes a request for my-remote-service.com, Cilium
 learns the IP address and will allow traffic due to the match on the name under
 the ``toFQDNs.matchName`` rule.
 
+.. note::  DNS rules do not currently support port ranges.
+
```

Comment:
Again in context, I think the note here calls out information that is not very relevant to what the reader is processing. The previous paragraph is talking about the example that comes immediately after this note, then there's the example after this, and in the middle is this note which is talking about something else.

DNS rules are L7 rules, and the note is under the L7 rules section as well, so we can probably drop this. If you feel we are not communicating the constraints around port ranges well enough, then I think we should collect all of these notes and put them as plain text under the port ranges section so that any user who attempts to use port ranges can look at the port ranges section and immediately understand how to use the feature and how not to use the feature.
```suggestion
```